exporter/sumologicexporter/exporter_test.go

```diff
@@ -58,3 +62,85 @@ func TestInitExporterInvalidCompressEncoding(t *testing.T) {
 
 	assert.EqualError(t, err, "unexpected compression encoding: test_format")
 }
+
+func TestAllSuccess(t *testing.T) {
+	test := prepareSenderTest(t, []func(res http.ResponseWriter, req *http.Request){
+		func(res http.ResponseWriter, req *http.Request) {
+			res.WriteHeader(200)
+			res.Write([]byte(""))
```

Comment:
>     // WriteHeader sends an HTTP response header with the provided
>     // status code.
>     //
>     // If WriteHeader is not called explicitly, the first call to Write
>     // will trigger an implicit WriteHeader(http.StatusOK).
>     // Thus explicit calls to WriteHeader are mainly used to
>     // send error codes.

Thus, `res.Write([]byte(""))` will be enough

You can also use e.g. `io.WriteString(res, "")`

BTW (nitpick): usually in Go, http handler signature looks like this `func(w http.ResponseWriter, req *http.Request)`, mark the first param called `w` not `res`

Reply:
I skip both lines as this behaves as expected